packages/utils/telemetry-utils/src/sampledTelemetryHelper.ts

```diff
@@ -82,11 +85,13 @@ export class SampledTelemetryHelper implements IDisposable {
 	 */
 	public constructor(
 		private readonly eventBase: ITelemetryGenericEvent,
-		private readonly logger: ITelemetryLoggerExt,
+		logger: ITelemetryBaseLogger,
```

Comment:
This class is actually internal so doesn't matter if we change it; but could also do the adjustment at the constructor call sites.